.github/labeler.yml

```diff
@@ -21,6 +21,11 @@
   - "**/*.dts"
   - "include/devicetree/*"
   - "include/dt-bindings/**/*"
+  - "scripts/dts/**/*"
+"area: Device Tree Binding":
+  - "dts/bindings/**/*"
+"area: Device Tree Tooling":
```

Comment:
While you're here, is there any chance of renaming these "Devicetree" instead? It's OK if that's out of scope for this PR.

Reply:
Added include/devicetree.h and renamed the labels.